server/service/src/demographic/insert_demographic_indicator.rs

```diff
@@ -56,8 +57,17 @@ pub fn validate(
     input: &InsertDemographicIndicator,
     connection: &StorageConnection,
 ) -> Result<(), InsertDemographicIndicatorError> {
-    if !check_year_name_combination_unique(&input.name, input.base_year, None, connection)? {
-        return Err(InsertDemographicIndicatorError::DemographicIndicatorAlreadyExistsForThisYear);
+    match &input.name {
```

Comment:
I don't really like combining validations because it reads confusing and isn't modular, but this seemed to work the best.